libshaderc_util/include/libshaderc_util/counting_includer.h

```diff
@@ -24,7 +24,7 @@ namespace shaderc_util {
 // An Includer that counts how many #include directives it saw.
 class CountingIncluder : public glslang::TShader::Includer {
  public:
-  CountingIncluder() : num_include_directives_(0) {}
+  CountingIncluder() : num_include_directives_({0}) {}
```

Comment:
This is fine because this atomic_int object is *created* by this constructor.  No race is possible until a client can see this object, which occurs after construction.
At first I thought would have to use in-class member initialization with ATOMIC_VAR_INIT macro call, but we don't need to go that far.